lib/src/fileset.rs

```diff
@@ -257,8 +252,7 @@ fn split_glob_path_i(input: &str) -> (&str, &str) {
         .take_while(|component| {
             !component
                 .trim_end_matches(path::is_separator)
```

Comment:
nit: you can remove `.trim_end_matches()`. It's not important to strip off path separators.